benefit/VERSION.yaml

```diff
@@ -1 +1 @@
-TAG: 5.15.2
+TAG: 5.15.3
```

Comment:
I think I am grabbing that value in my PR, so which ever of us gets approval second will have to move it further.